shared/common-adapters/icon.desktop.js

```diff
@@ -89,12 +90,15 @@ export function iconTypeToImgSet (type: IconType) {
   return `-webkit-image-set(${imgs})`
 }
 
-export function urlsToImgSet (imgMap: {[size: string]: string}): ?string {
-  const imgs = Object.keys(imgMap).map(size => `url('${imgMap[size]}') ${size}x`).join(', ')
-  if (!imgs) {
+export function urlsToImgSet (imgMap: {[size: string]: string}, targetSize: number): any {
+  const sizes = Object.keys(imgMap)
+
+  if (!sizes.length) {
     return null
   }
-  return `-webkit-image-set(${imgs})`
+
+  const str = sizes.map(size => `url('${imgMap[size]}') ${parseInt(size, 10) / targetSize}x`).join(', ')
```

Comment:
instead of the 'find the best 1x 2x definitions' code we instead give it fractional multipliers (1.25x etc) and the browser will do the right thing